src/main/resources/SLING-INF/config/sysusers/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended-org.redhat.pantheon.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<node>
+    <primaryNodeType>sling:OsgiConfig</primaryNodeType>
+    <property>
+        <name>user.default</name>
+        <value>pantheon</value>
+    </property>
+    <property>
+        <name>user.mapping</name>
+        <values>
+            <value>org.redhat.pantheon=pantheon</value>
+        </values>
+    </property>
+</node>
```

Comment:
Looks simple enough, but... what's with the gnarly filename? Is there some magic mapping/registration logic there?

Reply:
Yes, the file name is all convention based. It's basically:

`<class being condigured>.ammended-<symbolic bundle name>.xml`

or in this case

`org.apache.sling.serviceusermapping.imp.ServiceUserMapperImpl.amended-org.redhat.pantheon.xml`

Reply:
How on earth did you figure that out?